network/index.md

```diff
@@ -104,6 +103,30 @@ You can override these settings on a per-container basis.
 | `--dns-opt`    | A key-value pair representing a DNS option and its value. See your operating system's documentation for `resolv.conf` for valid options.                                                                                                                            |
 | `--hostname`   | The hostname a container uses for itself. Defaults to the container's ID if not specified.                                                                                                                                                                          |
 
+### Name resolution with multiple nameservers
+
+When you specify multiple DNS servers using `--dns` flags, name resolution may
+work in a surprising or unexpected way. DNS lookup behavior depends on a number
+of different factors:
```

Comment:
Ah right... We probably need to document how dns works with the default bridge. I think this could be done in a follow-up PR, to make @dvdksn life easier.

Reply:
After discussing with @akerouanton - this section can be removed because it simply isn't how hostname lookup works. Hostname lookup with the embedded nameserver and glibc is always done sequentially. There is a race condition that may occur with musl and nameservers using ipv6 addresses, which is documented in the following section.